docs/_docs/01_getting-started/quick-start.md

```diff
@@ -0,0 +1,310 @@
+---
+layout: collection-browser-doc
+title: Quick start
+category: getting-started
+excerpt: Learn how to start with Terratest.
+tags: ["quick-start"]
+order: 101
+nav_title: Documentation
+nav_title_link: /docs/
+---
+
+## Requirements
+
+Terratest uses the Go testing framework. To use Terratest, you need to install:
+
+- [Go](https://golang.org/) (requires version >=1.13)
+
+## Setting up your project
+
+The easiest way to get started with Terratest is to copy one of the examples and its corresponding tests from this
+repo. This quick start section uses a Terraform example, but check out the [Examples]({{site.baseurl}}/examples/) section for other
+types of infrastructure code you can test (e.g., Packer, Kubernetes, etc).
+
+1. Create an `examples` and `test` folder.
+
+1. Copy all the files from the [basic terraform example]({{site.baseurl}}/examples/code-examples/terraform-basic-example/) into the `examples` folder.
+
+1. Copy the [basic terraform example test]({{site.baseurl}}/examples/tests/terraform-basic-example-test/) into the `test` folder.
+
+1. To configure dependencies, run:
+
+    ```bash
+    cd test
+    go mod init "<MODULE_NAME>"
+    ```
+
+    Where `<MODULE_NAME>` is the name of your module, typically in the format
+    `github.com/<YOUR_USERNAME>/<YOUR_REPO_NAME>`.
+
+1. To run the tests:
+
+    ```bash
+    cd test
+    go test -v -timeout 30m
+    ```
+
+    *(See [Timeouts and logging]({{ site.baseurl }}/docs/testing-best-practices/timeouts-and-logging/) for why the `-timeout` parameter is used.)*
+
+
+## Terratest intro
+
+The basic usage pattern for writing automated tests with Terratest is to:
+
+1. Write tests using Go’s built-in [package testing](https://golang.org/pkg/testing/): you create a file ending in `_test.go` and run tests with the `go test` command. E.g., `go test my_test.go`.
+1. Use Terratest to execute your _real_ IaC tools (e.g., Terraform, Packer, etc.) to deploy _real_ infrastructure (e.g., servers) in a _real_ environment (e.g., AWS).
+1. Use the tools built into Terratest to validate that the infrastructure works correctly in that environment by making HTTP requests, API calls, SSH connections, etc.
+1. Undeploy everything at the end of the test.
+
+To make this sort of testing easier, Terratest provides a variety of helper functions and patterns for common infrastructure testing tasks, such as testing Terraform code, testing Packer templates, testing Docker images, executing commands on servers over SSH, making HTTP requests, working with AWS APIs, and so on.
+
+
+## Example #1: Terraform
+Let’s say you have the following (simplified) [Terraform](https://www.terraform.io/) code to deploy a web server in AWS (if you’re new to Terraform, check out our [Comprehensive Guide to Terraform](https://blog.gruntwork.io/a-comprehensive-guide-to-terraform-b3d32832baca)):
+
+```go

```

Comment:
I'm not sure what syntax highlighter we're using, or what theme, but it doesn't look great. For example, the comments in the code below render exactly the same as the code around them, rather than in a light-gray color that fades into the background:

<img width="818" alt="Screen Shot 2020-01-29 at 11 17 15 AM" src="https://user-images.githubusercontent.com/711908/73352405-f31d7180-4288-11ea-9988-ed9982fcf79a.png">

Is there any way to improve that?

Reply:
My bad, I set `golang`:
```
```go
```
instead of `hcl`. I fixed this.